client/comms/server.js

```diff
@@ -369,6 +370,7 @@ var Server = function (catalog_) {
   function respawnAllClients(){
     worldState.clients.forEach(function(client){
       respawn(client);
+      announceLoadout(client);
```

Comment:
why not put this in resetAllClients instead?
this function is for respawning it shouldn't do anything with loadouts

Reply:
Updated